Emby.Server.Implementations/Library/MusicManager.cs

```diff
@@ -28,8 +28,13 @@ public MusicManager(ILibraryManager libraryManager)
 
         public IReadOnlyList<BaseItem> GetInstantMixFromSong(Audio item, User? user, DtoOptions dtoOptions)
         {
-            return GetInstantMixFromGenres(item.Genres, user, dtoOptions);
-        }
+            var instantMixItems = GetInstantMixFromGenres(item.Genres, user, dtoOptions);
+
+            var result = new List<BaseItem> { item };
+            result.AddRange(instantMixItems.Where(i => !i.Id.Equals(item.Id)));
+
+            return result;
```

Comment:
```suggestion
            return [item, ..instantMixItems.Where(i => !i.Id.Equals(item.Id))];
```

Has the same compile output but in one line 🤷